core/execution/future/market.go

```diff
@@ -638,7 +640,19 @@ func (m *Market) Update(ctx context.Context, config *types.Market, oracleEngine
 		// market is settled, unsubscribe all
 		m.tradableInstrument.Instrument.Unsubscribe(ctx)
 	}
-
+	if tickSizeChanged {
+		peggedOrders := m.matching.GetActivePeggedOrderIDs()
+		peggedOrders = append(peggedOrders, m.peggedOrders.GetParkedIDs()...)
```

Comment:
Why get the parked orders here if we just continue if the order is not in the orderbook after?